src/views/Dissolution/DissolutionReviewConfirm.vue

```diff
@@ -39,8 +39,8 @@
           <section class="ml-5 mr-8 pl-2" v-if="isTypeCoop">
             <v-container id="dissolution-statement">
               <v-row no-gutters>
-                <v-col cols="3" class="inner-col-1">
-                  <label class="font-weight-bold">Dissolution<br>Statement</label>
+                <v-col cols="3" class="inner-col-1 pr-4">
```

Comment:
We removed the automatic gutters, so this is putting in a bit of a gutter so the first column (label) doesn't go right to the second column (data).